benchmarks/stdlib_bench_test.go

```diff
@@ -40,10 +40,12 @@ func BenchmarkStandardLibraryWithoutFields(b *testing.B) {
 }
 
 func BenchmarkZapStandardizeWithoutFields(b *testing.B) {
-	logger, err := zwrap.Standardize(zap.New(
-		zap.NewJSONEncoder(),
-		zap.DebugLevel,
-		zap.Output(zap.Discard)),
+	logger, err := zwrap.Standardize(
+		zap.New(
+			zap.NewJSONEncoder(),
```

Comment:
(as a separate change) We've duplicated these lines a lot in our benchmarks, and I think it's just adding noise to the tests, we should probably dedupe this with a helper.